### PR TITLE
FinVec is an instance of LeftModule

### PR DESCRIPTION
**Pull request by guilhermehas**

### CODE REVIEW

File: Cubical/Algebra/Module/Instances/VecFin.agda [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+open import Cubical.Reflection.RecordEquiv
+
+open import Cubical.Algebra.Monoid
+open import Cubical.Algebra.Ring
+open import Cubical.Algebra.Group
+open import Cubical.Algebra.Semigroup
+open import Cubical.Algebra.AbGroup
+open import Cubical.Algebra.Module
+
+module Cubical.Algebra.Module.Instances.VecFin {ℓ} (R : Ring ℓ) {n : ℕ} where
```
Comment (felixwellen) [inline Cubical/Algebra/Module/Instances/VecFin.agda:18]: Please make ```module Cubical.Algebra.Module.Instances.VecFin where``` the second line of the file and use an anonymous module here, to introduce the parameters, i.e. ```module _ {ℓ} (R : Ring ℓ) {n : ℕ} where``` (and intend what follows).

**Comment by felixwellen** [comment]
Thanks for the contribution, this is good to have!

File: Cubical/Algebra/Module/Instances/VecFin.agda [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  funExt λ _ → R'.+Assoc _ _ _
+·IdR (isMonoid (isGroup (+IsAbGroup (isLeftModule (snd FinVecLeftModule))))) _ = funExt λ _ → R'.+IdR _
+·IdL (isMonoid (isGroup (+IsAbGroup (isLeftModule (snd FinVecLeftModule))))) _ = funExt λ _ → R'.+IdL _
+·InvR (isGroup (+IsAbGroup (isLeftModule (snd FinVecLeftModule)))) _ = funExt λ _ → R'.+InvR _
+·InvL (isGroup (+IsAbGroup (isLeftModule (snd FinVecLeftModule)))) _ = funExt λ _ → R'.+InvL _
++Comm (+IsAbGroup (isLeftModule (snd FinVecLeftModule))) _ _ = funExt λ _ → R'.+Comm _ _
+⋆Assoc (isLeftModule (snd FinVecLeftModule)) _ _ _ = funExt λ _ → sym (R'.·Assoc _ _ _)
+⋆DistR+ (isLeftModule (snd FinVecLeftModule)) _ _ _ = funExt λ _ → ·DistR+ _ _ _
+⋆DistL+ (isLeftModule (snd FinVecLeftModule)) _ _ _ = funExt λ _ → ·DistL+ _ _ _
+⋆IdL (isLeftModule (snd FinVecLeftModule)) _ = funExt λ _ → R'.·IdL _
```
Comment (felixwellen) [inline Cubical/Algebra/Module/Instances/VecFin.agda:46]: You can make this part more readable by putting the (copattern) definition of the ```isLeftModule``` proof into a ```where```-block.

File: Cubical/Algebra/Module/Instances/VecFin.agda (hunk 1 above)
Comment (guilhermehas) [reply Cubical/Algebra/Module/Instances/VecFin.agda]: I have already fixed that.

File: Cubical/Algebra/Module/Instances/VecFin.agda (hunk 2 above)
Comment (guilhermehas) [reply Cubical/Algebra/Module/Instances/VecFin.agda]: I have already fixed that.

File: Cubical/Algebra/Module/Instances/VecFin.agda [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+module _ {ℓ} (R : Ring ℓ) {n : ℕ} where
+
+  open module R' = RingStr (snd R) renaming (_+_ to _+r_; -_ to -r_)
+  open IsMonoid using (isSemigroup; ·IdR; ·IdL)
+  open LeftModuleStr
+  open IsLeftModule
+  open IsSemigroup using (is-set; ·Assoc)
+  open IsGroup using (isMonoid; ·InvR; ·InvL)
+  open IsAbGroup
```
[the remark this answers is not in the document]
Comment (guilhermehas) [reply Cubical/Algebra/Module/Instances/VecFin.agda]: It was necessary before, but not anymore.
So I removed all unnecessary imports.

**Comment by felixwellen** [comment]
Ok, looks good! I'll merge it.

**Pull request merged**